backend/pyproject.toml

```diff
@@ -5,15 +5,15 @@ description = "Streamchaser's API"
 authors = ["Your Name <you@example.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.10"
+python = "^3.11"
 SQLAlchemy = "^1.4.31"
 requests = "^2.27.1"
 meilisearch = "^0.18.0"
-psycopg2-binary = "^2.9.3"
+psycopg2-binary = "^2.9.5"
 typer = "^0.4.0"
 tqdm = "^4.62.3"
 uvicorn = "^0.17.4"
-fastapi = "^0.73.0"
+fastapi = "^0.86.0"
 gunicorn = "^20.1.0"
 httpx = { version = "^0.22.0", extras = ["http2"] }
 pytest-asyncio = "^0.18.1"
```

Comment:
Daymn, did we only have 2 outdated ones?

Reply:
There is prob more, I just wanted to get dependabot up and running and do the job for me of checking :P

Reply:
Fair enough